tests/unit/store/createSubcollectionStore.ts

```diff
@@ -16,80 +32,71 @@ registerSuite({
 		}).then(dfd.resolve);
 	},
 
-	'should delegate to parent store': (function(){
-		function getStoreAndDfd(test: any) {
-			const dfd = test.async(1000);
-			const store = createSubcollectionStore<ItemType, CrudOptions>();
-			const Subcollection = store.createSubcollection();
-
-			return { dfd: dfd, store: store, Subcollection: Subcollection };
+	'should delegate to parent store': {
+		add(this: any) {
+			const { dfd, store, Subcollection } = getStoreAndDfd(this);
+			const data = createData();
+			Subcollection.add(data[0]);
+			store.fetch().then(function(results) {
+				assert.deepEqual(results, [ data[0] ], 'Didn\'t add item to source store');
+			}).then(dfd.resolve);
+		},
+
+		put(this: any) {
+			const { dfd, store, Subcollection } = getStoreAndDfd(this);
+			const data = createData();
+			Subcollection.put(data[0]);
+			store.fetch().then(function(results) {
```

Comment:
again, using `dfd` all over the place, this could have been 

``` javascript
return store.fetch().then((results) => {
   assert.deepEqual(results, [ data[0] ], 'Didn\'t put item in source store');
});
```